mpas_analysis/ocean/meridional_heat_transport.py

```diff
@@ -0,0 +1,213 @@
+    startDateClimo = config.get('climatology', 'startDate')
+    endDateClimo = config.get('climatology', 'endDate')
+    inputFilesClimo = historyStreams.readpath(streamName,
+                                              startDate=startDateClimo,
+                                              endDate=endDateClimo,
+                                              calendar=calendar)
+    simulationStartTime = get_simulation_start_time(runStreams)
+
+    print '\n  List of files for climatologies:\n' \
+          '    {} through\n    {}'.format(
+              os.path.basename(inputFilesClimo[0]),
+              os.path.basename(inputFilesClimo[-1]))
+
+    startYearClimo = config.getint('climatology', 'startYear')
+    endYearClimo = config.getint('climatology', 'endYear')
+    #   Create dictionary to store Climo related variables
+    dictClimo = {'inputFilesClimo': inputFilesClimo,
+                 'startDateClimo': startDateClimo,
+                 'endDateClimo':  endDateClimo,
+                 'startYearClimo': startYearClimo,
+                 'endYearClimo': endYearClimo}
+
+    sectionName = 'meridionalHeatTransport'
+
+    # Read in depth and MHT latitude points
+    # Latitude is from binBoundaryMerHeatTrans written in
+    #  mpaso.hist.am.meridionalHeatTransport.*.nc
+    # Depth is from refZMid, also in mpaso.hist.am.meridionalHeatTransport.*.nc
+    try:
+        mhtFile = historyStreams.readpath('meridionalHeatTransportOutput')[0]
+    except ValueError:
+        raise IOError('No MPAS-O MHT history file found: need at least one ')
+
+    print '  Read in depth and latitude...'
+    ncFile = netCDF4.Dataset(mhtFile, mode='r')
+    # reference depth [m]
+    refZMid = ncFile.variables['refZMid'][:]
+    binBoundaryMerHeatTrans = ncFile.variables['binBoundaryMerHeatTrans'][:]
+    binBoundaryMerHeatTrans = binBoundaryMerHeatTrans*rad_to_deg
+    ncFile.close()
+
+    ########################################################################
+    # Mark P Note: Currently only supports global MHT.
+    # Need to add variables merHeatTransLatRegion and merHeatTransLatZRegion
+    # these are not computed by default in ACME right now.
+    # Then we will need to add another section for regions with a loop
+    # over number of regions.
+    ########################################################################
+    variableList = ['avgMeridionalHeatTransportLat',
+                    'avgMeridionalHeatTransportLatZ']
+
+    print '\n  Compute and plot global meridional heat transport'
+
+    outputDirectory = build_config_full_path(config, 'output',
+                                             'mpasClimatologySubdirectory')
+
+    make_directories(outputDirectory)
+
+    outputFileClimo = '{}/meridionalHeatTransport_years{:04d}-{:04d}.nc'.format(
+                       outputDirectory, dictClimo['startYearClimo'],
+                       dictClimo['endYearClimo'])
+    if not os.path.exists(outputFileClimo):
```

Comment:
It never writes out the netcdf file right now, so the file always doesn't exist.